entry_tasks/16_Pick_movie_by _genre_or_actor/by_genre_and_cast.py

```diff
@@ -0,0 +1,119 @@
+def unique_cast(cast_values):
```

Comment:
Does python have some data type specifically useful if you need to have unique values? :)